hassio-google-drive-backup/backup/static/js/about.js

```diff
@@ -1,78 +1,113 @@
-    };
-    contributors.slice(0, contributorsCountToShow).forEach((contributor) => {
-      if (contributor.login in extra_info) {
-        const img = document.createElement("img");
-        img.src = contributor.avatar_url;
-        img.className = "contributor-img";
-        img.title = `@${contributor.login}`;
-        img.alt = img.title;
-
-        const a = document.createElement("a");
-        a.href = `https://github.com/${contributor.login}`;
-        a.target = "_blank";
-        a.appendChild(img);
-
-        const name = document.createElement("span");
-        name.innerHTML = `@${contributor.login}<br>${contributor.contributions} contributions`;
-        name.classList.add('contributor-name');
-        name.classList.add('center')
-        a.appendChild(name);
-
-        const div = document.createElement("div");
-        div.appendChild(a);
-
-        const detail = document.createElement("span");
-        detail.innerHTML = extra_info[contributor.login];
-        div.append(detail);
-        containerExtra.appendChild(div);
-        containerExtra.appendChild(document.createElement("br"));
-      } else {
-        const img = document.createElement("img");
-        img.src = contributor.avatar_url;
-        img.className = "contributor-img";
-        img.title = `@${contributor.login}`;
-        img.alt = img.title;
-
-        const a = document.createElement("a");
-        a.href = `https://github.com/${contributor.login}`;
-        a.target = "_blank";
-        a.appendChild(img);
-        containerSimple.appendChild(a);
-      }
+
+    const contributorsToShow = contributors.slice(0, CONTRIBUTORS_COUNT_TO_SHOW);
+    const contributorItems = await Promise.all(contributorsToShow.map((contributor) => loadContributor(contributor)));
+
+    contributorItems.forEach((item) => {
+      (item.isExtra ? containerExtra : containerSimple).appendChild(item.element);
     });
 
-    document.getElementById("contributors-more-link-count").innerHTML = contributors.length - contributorsCountToShow;
-    document.getElementById("contributors-count").innerHTML = contributors.length;
-    document.getElementById("contributors-count").classList.remove('default-hidden');
-    document.getElementById("contributors-loading").classList.add('default-hidden');
-  } else {
-    document.getElementById("contributors-loading").classList.add('default-hidden');
+    document.getElementById("contributors-more-link-count").innerText = contributors.length - CONTRIBUTORS_COUNT_TO_SHOW;
+    document.getElementById("contributors-count").innerText = contributors.length;
+    showElement("contributors-count");
+    showElement("contributors");
+    contributorsLoaded = true;
   }
+
+  hideElement("contributors-loading");
 }
 
 $(document).ready(function () {
-  M.Modal.init(document.querySelector("#about_modal"));
+  M.Modal.init(document.querySelector("#about_modal"), {
+    onOpenStart: () => fetchContributors(),
```

Comment:
No need to create a custom function `openAbout` for this; we can directly use the callback from the Materialize modal 😄

Reply:
Hrm, I should change all the other modals to use this too.